qt/widgets/instrumentview/src/InstrumentWidgetPickTab.cpp

```diff
@@ -100,6 +97,33 @@ InstrumentWidgetPickTab::InstrumentWidgetPickTab(InstrumentWidget *instrWidget)
   m_plot = new MiniPlot(this);
   connect(m_plot, SIGNAL(showContextMenu()), this, SLOT(plotContextMenu()));
 
+  // set up the rebin tab
+  m_rebin = new QWidget(this);
+  QGridLayout *rebinLayout = new QGridLayout(m_rebin);
+  m_rebinParams = new QLineEdit(m_rebin);
+  m_rebinParams->setToolTip("The parameters to use for the new binning, a comma separated list of first bin boundary, "
+                            "width, last bin boundary. See Rebin doc for details.");
+  m_rebinUseReverseLog = new QCheckBox("UseReverseLogarithmic", m_rebin);
```

Comment:
Please change the name of the checkbox to "Use reverse logarithmic" to be consistent in style with the other. You should keep, however, the name of the property as is.

Reply:
I was being consistent with Rebin parameters, but noted.

Reply:
Well but the other checkbox you don't put at all the actual property name. So either we take both property names as is from the Rebin, so (PreserveEvents and UseReverseLogarithmic), or if we rewrite our own labels, we should do for both to make them consistent.